modules/books.js

```diff
@@ -0,0 +1,81 @@
+export const titleInput = document.getElementById('title-input');
+export const authorInput = document.getElementById('author-input');
+export const submitButton = document.getElementById('submit-button');
+
+export class Methods {
+  constructor() {
+    this.books = [];
+  }
+
+  init = function () {
+    if (JSON.parse(localStorage.getItem('books')) !== null) {
+      this.books = JSON.parse(localStorage.getItem('books'));
+      this.books.forEach((element) => {
+        this.addBookToDocument(element);
+      });
+    }
+  };
+
+  removeBook = function (time) {
+    this.books = this.books.filter((book) => book.time !== time);
+    localStorage.setItem('books', JSON.stringify(this.books));
+  };
+
+  removeBtnEventListener = function (element, time) {
+    this.removeBook(time);
+    element.parentElement.remove();
+  };
+
+  addBookToDocument = function (book) {
+    const booksDiv = document.getElementById('books');
+
+    const bookDiv = document.createElement('div');
+    bookDiv.setAttribute('data-time', book.time);
+
+    const bookDesc = document.createElement('p');
+    bookDesc.classList.add('book-description');
+    const remove = document.createElement('button');
```

Comment:
- [ ] I would recommend using arrow function instead of using `function` keyword. It will make code clean and take less space as well.

As for example on the line 10 you can use like this 

`init = () => { ... }`

on line 19 you can use like this

`removeBook = (time) => { ... }`

Please update it.